creator-node/test/pollingTracks.test.js

```diff
@@ -559,8 +553,8 @@ describe('test Polling Tracks with real IPFS', function () {
       .expect(200)
   })
 
-  it('should not throw error response if saving metadata to ipfs fails', async function () {
-    sinon.stub(ipfs, 'add').rejects(new Error('ipfs add failed!'))
+  it.skip('should not throw error response if saving metadata to ipfs fails', async function () {
```

Comment:
flagging skipped test

Reply:
agh, so the reason why I skipped this is bc with the changes on how we use ipfs add and how we hit `/tracks/metadata` route, it is hard to mock imports. if the route were a component (which is will be a part of future work item with improving test structure), we can mock the import

Reply:
discussed offline - will hold off and revisit when refactoring test suite